crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -33,15 +40,51 @@ impl Default for Visibility {
 }
 
 /// Algorithmically-computed indication of whether an entity is visible and should be extracted for rendering
-#[derive(Component, Clone, Reflect, Debug)]
+#[derive(Component, Clone, Reflect, Debug, Eq, PartialEq, Default)]
 #[reflect(Component)]
 pub struct ComputedVisibility {
-    pub is_visible: bool,
+    is_visible_in_hierarchy: bool,
+    is_visible_in_view: bool,
```

Comment:
This does make the component a slight bit bigger, and 14/16 bits go unused. Could we use a set of bitflags with a u8 underlying this? It would make the component remain as small as it is now.

Reply:
I experimented with this and didn't see any perf wins. I do have that listed in follow up work though (see the pr description). Happy to swap in bitflags if we have a compelling reason.

Reply:
(probably after 0.8 at this point)

Reply:
(reducing memory usage is a compelling reason!)

Reply:
Memory usage is the primary one. Given how many visible entities there are, this would be a nice reduction.